src/main/cpp/util/path_windows.cc

```diff
@@ -131,16 +131,26 @@ std::string PathAsJvmFlag(const std::string& path) {
   // misinterpret them.
   // See https://github.com/bazelbuild/bazel/issues/2576 and
   // https://github.com/bazelbuild/bazel/issues/6098
-  std::string::size_type pos = 0;
-  while (true) {
-    pos = cpath.find_first_of("/\\", pos);
-    if (pos == std::string::npos) {
-      break;
+  size_t separators = 0;
+  for (const auto& c : cpath) {
+    if (c == '/' || c == '\\') {
```

Comment:
nit: `std::count` could be used to count these numbers, but I'm not at all sure if it would result in simpler code.

Reply:
Thanks for the idea! I'd need to use `count_if` and I think the resulting code would be neither more readable nor more efficient.

Reply:
That sentiment, I can heartily agree with! :)